core/src/port/mod.rs

```diff
@@ -19,11 +19,12 @@ use std::ptr;
 
 use anyhow::{bail, Result};
 
-const RSS_KEY_LEN: usize = 40;
+const RSS_KEY_LEN: usize = 52;
```

Comment:
Why this change? Will this impact anything in RSS (e.g., sink core, load-balancing key)?

Reply:
Also -- this needs to be wrapped in a feature flag. Not sure if it's a result of my NIC or DPDK version (I ran this on 21.08), but my runtime requires RSS key size 40.

Reply:
Ah, I forgot to add gate for these. In 23.11 this length is 52 and in 20.11 is 40. We should probably have some smoke test in CI to detect if a simple retina example can be executed successfully.

Reply:
Did you check if this new RSS key still properly performs symmetric RSS? I'm not entirely sure if it should be zero-padded or if continuing the byte pattern is correct.

Reply:
The size does not depend on the version but on the type of NIC, this patch will break some NICs, and enable some others. I remember I carried my diff for the SIGCOMM paper. The right length should be queried with rte_eth_dev... Something:)

Reply:
Are you sure? I was able to run retina with dpdk 20.08 on my NICs (Intel 710 and 810) with length 40 but it required 52 with dpdk 23.11 and the same NICs.

Reply:
In fstack they have the 40 by default, but if the NIC requires 52 they use the 52 one. The symmetrical can be shorten of course, it's just a matter of setting the right length. https://github.com/F-Stack/f-stack/blob/8c6094901e4a035bac8324a5cc590ea9b6a7eb35/lib/ff_dpdk_if.c#L663